src/SearchResults.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState, useEffect } from "react";
+import TableHeader from "./TableHeader";
+import TableData from "./TableData";
+
+function SearchResults() {
+  useEffect(() => {
+    console.log("hello react");
+    fetch("https://cyf-react.glitch.me")
+      .then(res => {
+        return res.json();
+      })
+      .then(data => {
+        return setBookings(data);
+      });
+  }, []);
+  const [bookings, setBookings] = useState([]);
```

Comment:
Makes a little more sense to declare your state and the setter function at the top of your functional component - as you will need them later on